src/train.py

```diff
@@ -90,15 +93,15 @@
             criterion,
             optimizer,
             device,
-            savepath=io_parameters.uid,
+            savepath=io_parameters.uid_save,
```

Comment:
In line 85. The network should be sent to device before parsing it to `train_segmentation`

Reply:
Addressed